docs/dev/dev_environment.md

```diff
@@ -13,7 +141,7 @@ Follow the directions below for the specific development environment that you ch
 
 ## Poetry
 
-Poetry is used in lieu of the "virtualenv" commands and is used for either environment. The virtual environment will provide most of the Python packages required to manage the development environment such as **Invoke**, but see the [Local Development Environment](#local-development-environment) section to see how to install Nautobot if you're going to be developing locally. To get started, run the following commands:
+Poetry is used in lieu of the "virtualenv" commands and is leveraged in both environments. The virtual environment will provide most of the Python packages required to manage the development environment such as **Invoke**, but see the [Local Development Environment](#local-poetry-development-environment) section to see how to install Nautobot if you're going to be developing locally. To get started, run the following commands:
```

Comment:
`provides most of the Python packages`? what doesn't it provide?

Reply:
Took me a bit to figure it out - it's Nautobot itself. It seems a bit funny the first time, but if you follow the link you'll see that for the non-dockerized setup you have to add nautobot as an extra to the project - and that's not included.

Reply:
I've refactored the section.